src/app/docs/tabs/tabs-docs.component.html

```diff
@@ -4,10 +4,373 @@
+        [tabHeaderCount]="demoSettings.showCounts ? tab.tabHeaderCount : null"
+        [tabHeading]="tab.tabHeading"
+      >
+        {{ tab.tabContent }}
+      </sky-tab>
+    </sky-tabset>
+
+    <sky-tabset *ngIf="demoSettings.includeAddAndClose"
+      (newTab)="onNewTabClick()"
+    >
+      <sky-tab *ngFor="let tab of tabArray; let i = index"
+        [tabHeaderCount]="demoSettings.showCounts ? tab.tabHeaderCount : null"
+        [tabHeading]="tab.tabHeading"
+        (close)="onCloseClick(i)"
+      >
+        {{ tab.tabContent }}
+      </sky-tab>
+    </sky-tabset>
+  </sky-docs-demo>
+
+  <sky-docs-code-examples>
+    <sky-docs-code-example
+      heading="Static tabs"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/tabs/static"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Dynamic tabs bound to an array"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/tabs/dynamic"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Static tabs with counts"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/tabs/counts"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Static tabs with add and close"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/tabs/static-add-close"
+    ></sky-docs-code-example>
+    <sky-docs-code-example
+      heading="Dynamic tabs with add and close"
+      sourceCodePath="src/app/public/plugin-resources/code-examples/tabs/dynamic-add-close"
+    ></sky-docs-code-example>
+  </sky-docs-code-examples>
+
   <sky-docs-design-guidelines>
-    See documentation for how to setup design guidelines.
+    
+      <sky-docs-demo-page-section
+      heading="Usage"
+    >
+      <sky-docs-design-guideline
+        heading="Use when"
+        headingStyle="success"
+        >
+        <p>
+          Use tabs when users require quick access to different slices of the same data.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use tabs to show difference slices of the same data."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/tabs/tabs-usage-1.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+          
+        </sky-docs-design-guideline>
+          
+        <sky-docs-design-guideline>
+            <p>
+                Use tabs when users need to complete different tasks on a page and each task includs a significant amount of data.
+            </p>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use tabs to group content for different tasks."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/tabs/tabs-usage-2.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+          
+        <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger"
+        >
+        <p>
```

Comment:
Looks like there is a .5 or 1px gap between the active tab and tab line in the Don't examples.

Reply:
Good catch. Fixing 5px gap.

Reply:
Looks to me like this was fixed. Can we resolve the comment?